inference-engine/src/inference_engine/src/ie_core.cpp

```diff
@@ -68,10 +68,12 @@ Parsed<T> parseDeviceNameIntoConfig(const std::string& deviceName, const std::ma
         if (deviceName.find("AUTO:") == 0) {
             config_[InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES] =
                 deviceName.substr(std::string("AUTO:").size());
+            config_.insert({CONFIG_KEY_INTERNAL(MULTI_WORK_MODE_AS_AUTO), InferenceEngine::PluginConfigParams::YES});
```

Comment:
if  you rebased on the recent master, after https://github.com/openvinotoolkit/openvino/pull/7310 is merged, I was thinking that this line is not needed. also below

Reply:
I fix some code issues on new PR [7380](https://github.com/openvinotoolkit/openvino/pull/7380) , this PR can't trigger jenkins CI， sorry for this inconvenience